NAMESPACE

```diff
@@ -1,9 +1,10 @@
 # Generated by roxygen2: do not edit by hand
 
+export(bar_plot_xctr)
```

Comment:
Use new name everywhere -- function names, file names etc. Although the content of some toy dataset may change, the rename is stable.

https://2degreesinvesting.github.io/tiltIndicator/reference/rename.html

If we introduce a new function with a deprecated name, it'll be a lot of work to change it later. If the function is exported then users will know about it and to rename it we need to go through an expensive deprecation process:

https://lifecycle.r-lib.org/articles/communicate.html#functions